src/screens/Forum/EditForumPostScreen.js

```diff
@@ -0,0 +1,213 @@
+import React, {
+  useState, useEffect, useCallback, useRef,
+} from 'react';
+import {
+  ScrollView, Picker, Alert, StyleSheet, View,
+} from 'react-native';
+import {
+  TextInput, Button, Text,
+} from 'react-native-paper';
+// import auth from '@react-native-firebase/auth';
+import firestore from '@react-native-firebase/firestore';
+import PropTypes from 'prop-types';
+import { theme } from '../../style';
+
+export default function EditForumPostScreen({ navigation }) {
+  const [title, setTitle] = useState('');
+  const [body, setBody] = useState('');
+  const [errorMessageTitle, setErrorMessageTitle] = useState(null);
+  const [errorMessageBody, setErrorMessageBody] = useState(null);
+  const [errorMessageFirestore, setErrorMessageFirestore] = useState(null);
+
+  const postID = navigation.getParam('postID');
+
+  const [categoryID, setCategoryID] = useState('');
```

Comment:
I think useState(null) works for strings, though I would double-check if I were you